### PR TITLE
Guide 2 or 3 depending on how you count

### PR DESCRIPTION
**Pull request by Grinch**
- Rewrite SimplePageView screen to better fit with newer look and feel
- Update SimplePageDetails and SimplePageCreate to use tab system
- Add WYSIWYGTextBox
- [UIContainer] Add setWidth, setHeight, getComponents, and setPadding(int) methods
- [UIMessageBox] Remove MessageBoxConsumer, was pointless
- [UITextBox] Add wrapText method to insert a prefix and suffix to the selected text
- Split Guide page names on " - " and display the first part as a header and the second part as a subheader
- Fix list hover state not always clearing
- Remove SimpleConfirmRemove screen
- Fix font colors on labels of child screens
- Fix alpha value of child screens
- Fix scroll/cursor position resetting at inappropriate times
- Cleanup scheduled task code
- Use UIConstants.DEFAULT_FO by default for UITextBox
- Switch between UIConstants.DEFAULT_FO and UIConstants.READ_ONLY_FO when switching editable state in UITextBox
- Cleanup unnessecary usages of UITextBox#setFontOptions due to above mention changes
- Finalize a lot of method parameters
- Various other changes

tl:dr; https://i.imgur.com/HMGfTTd.png

Signed-off-by: Steven Downer <grinch@outlook.com>

### CODE REVIEW

**Comment by Dockter** [comment]
Thank you for bringing Guide version 3.0 to the next level.
